GenerateSacrificeSettings.sh

```diff
@@ -0,0 +1,440 @@
+    # join
+    4)
+		echo "--join" >> settings.txt
+		# call the observer() function we declared at the bottom of the file
+		observer
+		;;
+	# 3v3
+	3)
+		echo "--host=6" >> settings.txt
+		echo "--map-list=maps-tg-3v3.txt" >> settings.txt
+		echo "--3v3                                        # host 3v3 (overrides team settings)" >> settings.txt
+		echo "--random-gods" >> settings.txt
+		echo "--shuffle-sides" >> settings.txt
+		echo "--stutter-on-desynch" >> settings.txt
+		echo "--no-particles" >> settings.txt
+		;;
+	# 2v2
+	2)
+		echo "--host=4" >> settings.txt
+		echo "--2v2                                        # host 2v2 (overrides team settings)" >> settings.txt
+		echo "--random-gods" >> settings.txt
+		echo "--shuffle-sides" >> settings.txt
+		echo "--stutter-on-desynch" >> settings.txt
+		
+		# another map selection ...
+		echo "1. shiny's 2v2 map list"
+		echo "2. tg's 2v2 map list"
+		echo "3. tree's 2v2 map list"
+	
+		maplist=""
+		while [[ ! $maplist =~ ^[1-3]$ ]]; do
+			read -p "Enter a number between 1 and 3: " maplist
+			if [[ ! $maplist =~ ^[1-3]$ ]]; then
+				print_error_message
+			fi
+		done
+		case $maplist in
+			1) # shiny 2v2
+				echo "--map-list=maps-shiny-2v2.txt" >> settings.txt
+				;;
+			2) # tg 2v2
+				echo "--map-list=maps-tg-2v2.txt" >> settings.txt
+				;;
+			3) # tree 2v2
+				echo "--map-list=maps-tree-2v2.txt" >> settings.txt
+				;;
+		esac
+		;;
+	# 1v1
+	1)
+		echo --host=2 >> settings.txt
+		echo --random-gods >> settings.txt
+		echo --shuffle-sides >> settings.txt
+		echo --stutter-on-desynch >> settings.txt
+
+		# another 1v1 map selection...
+		echo "1. shiny's 1v1 map list"
+		echo "2. tg's 1v1 map list"
+		echo "3. tree's 1v1 map list"
+		echo "4. (2) Ferry.scp"
+		maplist=""
+		while [[ ! $maplist =~ ^[1-4]$ ]]; do
+			read -p "Enter a number between 1 and 4: " maplist
+			if [[ ! $maplist =~ ^[1-4]$ ]]; then
+				print_error_message
+			fi
+		done
+		case $maplist in
+			1) # shiny 1v1
+				echo "--map-list=maps-shiny-1v1.txt" >> settings.txt
+				;;
+			2) # tg 1v1
+				echo "--map-list=maps-tg-1v1.txt" >> settings.txt
+				;;
+			3) # tree 1v1
+				echo "--map-list=maps-tree-1v1.txt" >> settings.txt
+				;;
+		esac
+		;;
+	
+esac
+
+
+# Prompt for enabling Slaughter
+read -p "Enable Slaughter (Y/N): " slaughter_choice
+if [[ $slaughter_choice =~ ^[Yy]$ ]]; then
+    echo "Choose the number of kills required to win:"
+    echo "1. 100 kills"
+    echo "2. 250 kills"
+    echo "3. 500 kills"
+    echo "4. 999 kills"
+    read -p "Enter the number corresponding to the desired option: " slaughter_kills
+    case $slaughter_kills in
+        4) echo "--slaughter=999" >> settings.txt ;;
+        3) echo "--slaughter=500" >> settings.txt ;;
+        2) echo "--slaughter=250" >> settings.txt ;;
+        1) echo "--slaughter=100" >> settings.txt ;;
+    esac
+else
+    echo "# --slaughter" >> settings.txt
+fi
+# with everything done, call the observer() function located at the end of the file
```

Comment:
same comment about this comment being wrong about where teh functions are located.

Reply:
fixed.